packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -1156,18 +1159,10 @@ export function makeWallet({
     await paymentRecord.actions.deposit(depositTo);
   };
 
-  // Allow people to send us payments.
-  const selfContact = addContact(
-    'Self',
-    Far('contact', {
-      receive(payment) {
-        return addPayment(payment);
-      },
-    }),
-  );
+  let selfContactP;
   async function getDepositFacetId(_brandBoardId) {
     // Always return the generic deposit facet.
-    return E.get(selfContact).depositBoardId;
+    return E.get(selfContactP).depositBoardId;
```

Comment:
It's not entirely obvious that `selfContactP` is guaranteed to be defined by the time we get here. Are we guaranteed that `initialize` gets called before this? I expect an `assert` would be requirted for this to `@ts-check`.

Reply:
We are guaranteed that `initialize' is called.  It probably doesn't need the `assert` just because of the untyped declaration (default `any`).